data/Viveksati5143/events/2023-06-08-AWSOME-DAY-Conference.json

```diff
@@ -0,0 +1,17 @@
+{
+  "isVirtual": true,
+  "isInPerson": true,
```

Comment:
Will it held `inPerson` too!? Asking because I couldn't find any info about it, if not, Please remove this key from your json. 

```suggestion
```